IMC/chatbot/Chatbot.md

```diff
@@ -0,0 +1,170 @@
+# Covid-19 and ChatBots
+
+![img](images/1.jpg)
+
+## What is a Chatbot?
+
+A Chatbot is a software that you can “chat with.” You can find Chatbots in messaging apps such as Facebook Messenger, or on websites through pop-ups. They can also be integrated with “virtual assistants” for customer service or tech support. Google Assistant and Amazon Alexa are both great examples.
+
+With the rapid improvements in Natural Language Processing (NLP), it has never been easier to create a chatbot. If you pay  attention to the structures of modern websites, you will notice that most of them use an existing chatbot integrated with the web pages. These bots are considered smart chatbots as they have the ability to understand language structures. Although limited in general, they provide all the necessary components of a virtual assistant to help you navigate a website. Let's see how easy is it to create a chatbot!
```

Comment:
should chatbot be capitalized or not? Sometimes it is and sometimes it isn't so it's best to pick one and stick with it

Reply:
I changed all to be capitalized